index.js

```diff
@@ -0,0 +1,123 @@
+const index = document.querySelector("#index");
+const number = document.querySelector(".number");
+const error50 = document.querySelector(".error50");
+const spinner = document.querySelector("#spinner");
+const spinner2 = document.querySelector("#spinner2");
+const errorMessage = document.querySelector("#error");
+const results = document.querySelector(".result");
+const check = document.querySelector("#accept");
+const selector = document.querySelectorAll(".dropdown-item");
+const selected = document.querySelector(".mySelect");
+const button = document.querySelector(".btn");
+
+//button Listener
+button.addEventListener("click", async () => {
+  number.classList.add("d-none");
+  index.classList.remove("is-invalid");
+  error50.classList.add("d-none");
```

Comment:
You could have made this hiding / showing functionality into a function, since you are repeating this across your code a lot. It would have been more DRY